### DISCUSSION TITLE
Config

### DISCUSSION THREAD

**Discussion by thagoat** [Dark-Alex-17/managarr#28] [General]
Ok, what am I missing? My config.yml says null after both radarr and sonarr, but I can't seem to put the right info in the file. I just get errors. Any help would be appreciated. Feeling like a dingbat lol

**Comment by Dark-Alex-17** [comment]
Hey no worries I'm happy to help! Mind if I ask what your config file looks like currently? With the API key redacted, of course.

**Comment by thagoat** [comment]
<code>radarr: null
sonarr: null</code>

**Reply by Dark-Alex-17** [reply]
Ah okay now I see what you meant. Okay, so just so I can know how to help you best and not make any assumptions, how much experience do you have home-labbing, or with Radarr and Sonarr?

**Reply by thagoat** [reply]
Been self-hosting for 15 years, pretty knowledgeable lol, I just don't know what to put in the yml file, API key obviously and ip:port, I just know know the syntax. I assume 
<code>radarr:
                - host: "ip:port"
                - api: "key"</code>
Is that about right?

**Comment by Dark-Alex-17** [comment]
Ah okay! Sorry, I'm being real slow today!

There's an [example config file in the README](https://github.com/Dark-Alex-17/managarr?tab=readme-ov-file#example-configuration) that you can refer to for formatting!

Let me know if that helps and works for you!

**Reply by thagoat** [reply]
Perfect, I didn't see that for the life of me when I posted here. Thank you very much!

**Reply by Dark-Alex-17** [reply]
No worries! We all make mistakes! Do you mind if I close this discussion?

**Reply by thagoat** [reply]
Working now, thanks!
